refactron/core/pipeline.py

```
@@ -0,0 +1,62 @@
+            AnalysisResult containing issues and metrics.
+        """
+        target_path = Path(target)
+        
+        # 1. Resolve config the same way as CLI
+        if self.config_path and self.config_path.exists():
+            config = RefactronConfig.from_file(self.config_path)
+        else:
+            if not self.project_root:
+                try:
+                    self.project_root = Refactron().detect_project_root(target_path)
+                except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)